src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -987,12 +987,9 @@ public boolean isFeedbackSessionHasQuestionForStudents(
         return !allQuestions.isEmpty();
     }
 
-    public boolean isFeedbackSessionCompletedByStudent(
-            String feedbackSessionName,
-            String courseId, String userEmail)
-            throws EntityDoesNotExistException {
-
-        FeedbackSessionAttributes  fsa = this.getFeedbackSession(feedbackSessionName, courseId);
+    public boolean isFeedbackSessionCompletedByStudent(FeedbackSessionAttributes fsa,
+                                                       String userEmail)
+                   throws EntityDoesNotExistException {
         if (fsa == null) {
```

Comment:
Since you are getting the `fsa` as a parameter, it's reasonable to expect it to be not-null. Can use an assertion instead.